constraints.txt

```diff
@@ -38,7 +38,7 @@ babel==2.9.1
     # via sphinx
 backcall==0.2.0
     # via ipython
-backports.entry-points-selectable==1.1.1
+backports-entry-points-selectable==1.1.1
```

Comment:
Any idea why these changed?

Reply:
No, and a quick Google search tells me something is probably amiss here. Will investigate.

Reply:
For now I just set this back.